regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -85,6 +85,13 @@ var PreambleView = ChildView.extend({
 
   write: function(section, tocId, label, $parent) {
     this.mode = 'write';
+
+    // Top-level sections and permalink sections use different markup;
+    // find appropriate element on top-level sections. TODO: unify markup
+    // and/or fetch excerpts asynchronously
```

Comment:
I think I understand the intent here, but I don't understand these terms. Could you provide an example?